ambry-server/src/test/java/com.github.ambry.server/ServerPlaintextTokenTest.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.server;
+
+import com.github.ambry.clustermap.DataNodeId;
+import com.github.ambry.network.Port;
+import com.github.ambry.network.PortType;
+import com.github.ambry.utils.SystemTime;
+import com.github.ambry.utils.Utils;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+
+/**
+ * The multi node single partition test needs to run with clean stores as it checks replication token offset values
+ * so it has been put into a separate class with per-test initialization and cleanup
+ */
+public class ServerPlaintextTokenTest {
+  private Properties coordinatorProps;
+  private MockNotificationSystem notificationSystem;
+  private MockCluster plaintextCluster;
+
+  @Before
+  public void initializeTests()
+      throws Exception {
+    coordinatorProps = new Properties();
+    notificationSystem = new MockNotificationSystem(9);
+    plaintextCluster =
+        new MockCluster(notificationSystem, SystemTime.getInstance());
+    plaintextCluster.startServers();
+  }
+
+  public ServerPlaintextTokenTest()
+      throws Exception {
+  }
+  @After
+  public void cleanup() {
+    long start = System.currentTimeMillis();
```

Comment:
I know this is from before, but in your tests now do you notice this? If not, let us remove this.